Sources/Basic/OutputByteStream.swift

```diff
@@ -230,6 +237,51 @@ public class _OutputByteStreamBase: OutputByteStream {
         }
         buffer += bytes
     }
+
+    /// Write an ArraySlice of bytes to the buffer.
+    public final func write(_ bytes: ArraySlice<UInt8>, buffered: Bool = true) {
+        guard buffered else {
+            writeImpl(bytes)
```

Comment:
This should probably flush, if you are going to put the buffered annotation on the write method.

It might be better to put the buffered flag on the entire stream, so then no buffer is ever used and we can avoid the flush here.